src/script/build-with-container.py

```diff
@@ -685,7 +712,8 @@ def bc_run_tests(ctx):
 
 @Builder.set(Steps.SOURCE_RPM)
 def bc_make_source_rpm(ctx):
-    """Build SPRMs."""
+    """Build SRPMs."""
```

Comment:
You would not believe how long I had to stare at this line to see what was different. :-P